src/common/ConfigManager.cpp

```diff
@@ -718,7 +718,11 @@ const char* FindConfigFile(const char *name)
 void LoadConfigFile()
 {
 #if !defined(_WIN32) && !defined(__APPLE__)

```

Comment:
Since your functions are cross-platform, remove this `#if` and just use the same code for all platforms.

Reply:
Also win32 has `stat()`:

https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/stat-functions?view=vs-2017

You may need something like:

```cpp
#ifdef _WIN32
    #define stat _stat
#endif 
```